pkg/relayer/proxy/relay_verifier.go

```diff
@@ -6,7 +6,10 @@ import (
 	sdkerrors "cosmossdk.io/errors"
```

Comment:
Leaving a comment here but it is appropriate elsewhere.

Can you write out an E2E test (in englih/comments) on how we can test & verify session rollover?

Just write out the steps (that do not compile) and add a TODO to add it in a followup PR.